packages/amplify-graphql-transformer-core/src/transformer-context/index.ts

```diff
@@ -29,6 +29,7 @@ import { ResolverManager } from './resolver';
 import { TransformerResourceHelper } from './resource-helper';
 import { StackManager } from './stack-manager';
 import { assetManager } from './asset-manager';
+import { DataSource } from 'aws-cdk-lib/aws-stepfunctions-tasks';
```

Comment:
Huh. Not sure where this came from...I'm assuming I was intending to import the `DatasourceType` utility type and VSCode tried to be helpful. Will remove or replace with correct type.

Reply:
Resolved in a82f5a05e